plugins/UsersManager/API.php

```diff
@@ -513,6 +514,22 @@ public function getUsersHavingSuperUserAccess()
     }
 
     /**
+     * Regenerate the token_auth associated with a user.
```

Comment:
We should add a comment that the same auth token cannot be used afterwards if $userLogin is the own login.